modules/apps/object/object-test/src/testFunctional/tests/ObjectExportImport.testcase

```diff
@@ -179,13 +179,7 @@ definition {
 		}
 
 		task ("And given the object definition is imported") {
-			ObjectExportImport.importObject(
-				objectFileName = "ImportedObjectWith100Fields",
-				objectName = "CustomObject187390");
-
-			ObjectPortlet.selectCustomObject(label = "Custom Object 187390");
-
-			ObjectAdmin.publishObjectViaUI();
```

Comment:
After the Modifiable story, objects can be imported already published, making this step unnecessary.